rllib/algorithms/algorithm.py

```diff
@@ -819,6 +874,8 @@ def duration_fn(num_units_done):
                         num_units_done += (
                             _agent_steps if self._by_agent_steps else _env_steps
                         )
+                    if self.reward_estimators:
+                        all_batches = all_batches.extend(batches)
```

Comment:
so quick question -- how do we make sure that all batches contains that correct amount/ correct data for doing off policy eval -- e.g. how do I make sure that all batches has my whole offline dataset?

Reply:
Currently, you need to know the size of the batch up front in either timesteps or episodes, and pass it to `evaluation_duration` up front.